src dir missing

index.js, line 9: `module.exports = require('./src/web-router');`

the 'src' directory is not included in either the git repository or the module on npm. as a result, any use of the module yields this error: `Error: Cannot find module './src/web-router'`

changing the line to `module.exports = require('./lib/web-router');` fixes the problem.

--- a/index.js
+++ b/index.js
@@ -6,4 +6,4 @@
 
 'use strict';
 
-module.exports = require('./src/web-router');
+module.exports = require('./lib/web-router');

